geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/sortedset/AbstractZAddIntegrationTest.java

```diff
@@ -301,6 +306,106 @@ public void shouldUpdateScore_whenSettingMemberThatAlreadyExists() {
     assertThat(jedis.zscore(key, member)).isEqualTo(1.0);
   }
 
+
+  @Test
+  public void zaddIncrOptionSupportsOnlyOneIncrementingElementPair() {
+    assertThatThrownBy(() -> jedis.sendCommand(SORTED_SET_KEY, Protocol.Command.ZADD,
+        SORTED_SET_KEY, incrOption, "1", member, "2", "member_1"))
+            .hasMessageContaining(RedisConstants.ERROR_ZADD_OPTION_TOO_MANY_INCR_PAIR);
+  }
+
+  @Test
+  public void zaddIncrOptionThrowsIfIncorrectScorePair() {
+    assertThatThrownBy(() -> jedis.sendCommand(SORTED_SET_KEY, Protocol.Command.ZADD,
+        SORTED_SET_KEY, incrOption, "1", member, "2")).hasMessageContaining(ERROR_SYNTAX);
+  }
+
+  @Test
+  public void zaddIncrOptionDoseNotAddANewMemberWithXXOption() {
+    Object result = jedis.sendCommand(SORTED_SET_KEY, Protocol.Command.ZADD, SORTED_SET_KEY, "XX",
+        incrOption, "1", member);
+    assertThat(result).isNull();
+    assertThat(jedis.zscore(SORTED_SET_KEY, member)).isNull();
+  }
+
+  private final double initial = 355.681000005;
+  private final double increment = 9554257.921450001;
+  private final double expected = initial + increment;
```

Comment:
Could these fields be moved to the top of the class for consistency with other classes please?